crates/bevy_render/src/renderer/render_device.rs

```diff
@@ -82,8 +85,17 @@ impl RenderDevice {
 
     /// Creates a new [`BindGroup`](wgpu::BindGroup).
     #[inline]
-    pub fn create_bind_group(&self, desc: &wgpu::BindGroupDescriptor) -> BindGroup {
-        let wgpu_bind_group = self.device.create_bind_group(desc);
+    pub fn create_bind_group<'a>(
+        &self,
+        label: Label<'a>,
```

Comment:
Would be nice if this could take a `&str` directly to avoid all the `Some("")` everywhere. Is it possible to maybe have a `IntoLabel` trait implemented for `&str`?

I wouldn't block the PR on that though. Just something that could be nice too.